CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ### Fixes
 
+- **AWS SQS Scaler**: Respect `scaleOnDelayed` value ([#4377](https://github.com/kedacore/keda/issue/4377))
```

Comment:
Could you move this to `New` section? This isn't a fix because it's a new feature. The message should be something like `Add support to scaledOnDelayed`